README.md

```diff
@@ -44,31 +44,46 @@ console.log(countries.getNames("en")); // { 'AF': 'Afghanistan', 'AL': 'Albania'
 
 * `ar`: Arabic
 * `az`: Azerbaijani
+* `be`: Belorussian
+* `bg`: Bulgarian
+* `bs`: Bosnian
 * `cs`: Czech
 * `da`: Danish
 * `de`: German
 * `en`: English
 * `es`: Spanish
 * `et`: Estonian
+* `fa`: Persian
 * `fi`: Finnish
 * `fr`: French
 * `el`: Greek
 * `he`: Hebrew
+* `hr`: Croatian
 * `hu`: Hungarian
+* `hy`: Armenian
 * `it`: Italian
 * `ja`: Japanese
 * `ka`: Georgian
 * `kk`: Kazakh
 * `ko`: Korean
+* `ky`: Kyrgyz
+* `lt`: Lithuanian
+* `lv`: Latvian
+* `mk`: Macedonian
+* `mn`: Mongolian
 * `nb`: Norwegian Bokmål
 * `nl`: Dutch
 * `nn`: Norwegian Nynorsk
 * `pl`: Polish
 * `pt`: Portuguese
+* `ro`: Romanian
 * `ru`: Russian
+* `sk`: Slovak
```

Comment:
sl is missing here?

Reply:
Nope, Slovak is sk. https://en.wikipedia.org/wiki/Slovak_language
sl is for Slovene https://en.wikipedia.org/wiki/Slovene_language

Reply:
I will fix that, now too :) good catch